From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nitish Sakhawalkar <nitsakh@icloud.com>
Date: Thu, 20 Sep 2018 17:47:58 -0700
Subject: v8_context_snapshot_generator.patch

v8_context_snapshot_generator is a build time executable.
The patch adds the config.

diff --git a/tools/v8_context_snapshot/BUILD.gn b/tools/v8_context_snapshot/BUILD.gn
index b119ec095c7b1b55277a8d6784be35508ac37094..47df8af5ef68bf8e3fb4fa4f22f7d008c4921dd5 100644
--- a/tools/v8_context_snapshot/BUILD.gn
+++ b/tools/v8_context_snapshot/BUILD.gn
@@ -114,6 +114,7 @@ if (use_v8_context_snapshot) {
     configs += [
       "//v8:external_startup_data",
       ":disable_icf",
+      "//electron/build/config:build_time_executable"
     ]
   }
 }
